charts/datadog/templates/_containers-init-linux.yaml

```
@@ -53,6 +53,9 @@
       subPath: system-probe.yaml
       readOnly: true
     {{- end }}
+{{- if .Values.agents.containers.initContainers.volumeMounts }}
+{{ toYaml .Values.agents.containers.initContainers.volumeMounts | indent 6 }}
```

```suggestion
{{ toYaml .Values.agents.containers.initContainers.volumeMounts | indent 4 }}
```
Could you please double-check indentation? I was unable to render the template with this indentation. For improved readability consider putting whole block at indent 4 (and using `nindent 4`).